src/Npgsql/TypeHandlers/LqueryHandler.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Data;
+using System.IO;
+using System.Threading.Tasks;
+using Npgsql.BackendMessages;
+using Npgsql.PostgresTypes;
+using Npgsql.TypeHandling;
+using Npgsql.TypeMapping;
+using NpgsqlTypes;
+
+namespace Npgsql.TypeHandlers
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Yeah, ideally this wouldn't be here. Definitely no need to document private/internal stuff though.